tests/functional/s3/test_cp_command.py

```diff
@@ -393,6 +393,21 @@ def test_turn_off_glacier_warnings(self):
         self.assertEqual(self.operations_called[0][0].name, 'HeadObject')
         self.assertEqual('', stderr)
 
+    def test_turn_off_glacier_warnings_for_deep_archive(self):
```

Comment:
We have added tests for the negative cases. What do you think about adding tests for the positive cases? I see there are a lot of warning tests for ensuring that a warning is present for `GLACIER`. To get full testing parity, we should probably add these for `DEEP_ARCHIVE` as well.

Reply:
Sounds reasonable.